src/ray/protobuf/usage.proto

```diff
@@ -130,4 +130,12 @@ enum TagKey {
   // Name of Tune scheduler algorithm or "Custom" if user-defined.
   // Example: "FIFOScheduler"
   TUNE_SCHEDULER = 502;
+  // (AIR) Fully user-controlled experiment tracking integrations ("0" or "1")
+  // NOTE: These tags + the callback metrics can be aggregated to extract
+  // total experiment tracking integration usage.
+  AIR_SETUP_WANDB_INTEGRATION_USED = 503;
+  AIR_SETUP_MLFLOW_INTEGRATION_USED = 504;
```

Comment:
@jjyao does this pattern look reasonable to you?

Reply:
Yea, good to me